frontend/src/actions/googleAuthActions.js

```diff
@@ -0,0 +1,72 @@
+import URLSearchParams from 'url-search-params';
+
+export const url = 'http://127.0.0.1:8000';
+
+/*These are the django client ID and SECRET
+  from the OauthToolkit Application registered in your django admin
+*/
+export const django_client_id = 'FMg1tMRE2b7XzzdY3K7cvE6zNw6nwDSr5asPfyuN';
+export const django_client_secret =
```

Comment:
You can't embed secrets in the source. This needs to be moved out of the client.